packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  h,
+  Prop,
+  State,
+} from '@stencil/core';
+
+@Component({
+  tag: 'va-breadcrumbs',
+  styleUrl: 'va-breadcrumbs.css',
+  shadow: true,
+})
+export class VaBreadcrumbs {
+  @Element() el: HTMLElement;
+
+  /**
+   * Adds an aria-label attribute to the <nav /> element.
+   */
+  @Prop() label: string = 'Breadcrumb';
+
+  /**
+   * Analytics tracking function(s) will not be called
+   */
+  @Prop() disableAnalytics: boolean;
+
+  @State() breadcrumbs: Array<Node>;
+
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  private getClickLevel(target: HTMLAnchorElement) {
+    const anchorNodes = Array.from(this.el.querySelectorAll('a'));
```

Comment:
Do we need to have `this.el.querySelectorAll('a')` here or could we rely on the already computed `this.breadcrumbs` state variable?

Reply:
The first one is `this.el.querySelectorAll('a')` and the second array is `this.breadcrumbs`. I went with the querySelector because it made it easier to compare to the target.

<img width="738" alt="Screen Shot 2022-01-05 at 11 55 41" src="https://user-images.githubusercontent.com/36863582/148257231-0553c613-89c5-4343-97a4-abf179c8fcc7.png">
<img width="557" alt="Screen Shot 2022-01-05 at 11 55 52" src="https://user-images.githubusercontent.com/36863582/148257236-ef171c5e-fcae-4d7d-8b9f-53ade57c810b.png">